packages/server-wallet/src/engine/engine.ts

```diff
@@ -733,13 +733,20 @@ export class SingleThreadedEngine
    * @param channelId - The id of the channel to try and close.
    * @returns A promise that resolves to the channel output.
    */
-  async closeChannel({channelId}: CloseChannelParams): Promise<SingleChannelOutput> {
+  async closeChannel({
+    channelId,
+  }: CloseChannelParams): Promise<SingleChannelOutput & {newObjective: WalletObjective}> {
```

Comment:
Should you update the docstring above given the new shape of the return value?

Reply:
Yup, although this reminds me that we're still using the `engine` API to generate our docs. We should be using the `wallet` interface instead since that is now the externally facing API.